### PR TITLE
[bitnami/supabase] FIX: Public URL

### PR DESCRIPTION
**Pull request by jsmueller7**
### Description of the change

As described [here](https://github.com/bitnami/charts/issues/19583#issue-1915967951)

### Benefits

If one doesn't enable the kong ingress the public URL will be recognized through setting the `Values.global.publicURL` field.

### Possible drawbacks

none.

### Applicable issues

https://github.com/bitnami/charts/issues/19583#issue-1915967951

### Additional information

It's a very plain and simple thing.

### Checklist

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [ ] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [ ] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [ ] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by jsmueller7 — COMMENTED** [review]
This adds some additional values to realtime.

1. Realtime needs to know the database user in order to write data into another scheme (realtime). Otherwise the data will be written to the public scheme.
2. Without the `DNS_NODES` env realtime won't work in a cluster.

**Comment by javsalgar** [comment]
Hi!

Thank you so much for the PR! Could you fix the DCO issue and the merge conflict?

File: bitnami/supabase/templates/_helpers.tpl [hunk 1]
```diff
@@ -22,8 +22,8 @@ Return the proper Supabase Studio Public URL
 Return the proper Supabase API Public URL
 */}}
 {{- define "supabase.api.publicURL" -}}
-{{- if .Values.publicURL -}}
-{{- print .Values.publicURL -}}
+{{- if .Values.global.publicURL -}}
```
Comment (aoterolorenzo) [inline bitnami/supabase/templates/_helpers.tpl:25]: Shouldn't be at least a binding `.Values.global.publicURL` value in the _values.yaml_ file?

File: bitnami/supabase/values.yaml [hunk 2]
```diff
@@ -950,6 +950,7 @@ realtime:
     DB_HOST: {{ include "supabase.database.host" . | quote }}
     DB_PORT: {{ include "supabase.database.port" . | quote }}
     DB_NAME: {{ include "supabase.database.name" . | quote }}
+    DB_USER: {{ include "supabase.database.user" . | quote }}
```
Comment (aoterolorenzo) [inline bitnami/supabase/values.yaml:953]: What's the motivation of this changes? Seems to go further than just fix the public URL. Maybe unintentional changes?

Comment (jsmueller7) [reply bitnami/supabase/values.yaml:953]: These changes aren't connected to the publicURL issue. I should have opend a pull request on it's own for this.

**The background of these change suggestions is:**
* Without knowing the `DB_USER` the realtime extension isn't able to write into the schema of it's extension (wheather it's called "realtime" or "_realtime" - in your deployment there exists at least a schema "realtime"). Instead realtime then writes it's information into the public schema, wich isn't what you want.
* After adding `DB_USER` you can add `DB_AFTER_CONNECT_QUERY: 'SET search_path TO <schema the extension shall use>'. Then it won't be using the public schema any more.
* Without specifying the `DNS_NODES` variable realtime can't be run it in a cluster.
* `ENABLE_TAILSCALE: "false"` actually shouldn't change anything because it sould be the default behavior.

File: bitnami/supabase/templates/_helpers.tpl (hunk 1 above)
Comment (jsmueller7) [reply bitnami/supabase/templates/_helpers.tpl:25]: You're right. You can reject this request. I had a missconfig in my own local values.
